pages/design/atomic-template-empty.md

```diff
@@ -1,35 +1,38 @@
 ---
+id: 
 name:
 rule_type: atomic
 
 description: |
   This rule checks ...
 
-success_criterion:
-  - x.x.x # (Name of Success Criterion)
+accessibility_requirements:
+  wcag20:x.x.x: # (Name of Success Criterion)
```

Comment:
Should we have other examples, e.g. a technique?
I guess that most rules that test for a SC also test for a technique. So far we have listed it under Background, but it would make sense to have it as structured data, when possible.